upgrader/upgrade-061-070/mongodb-upgrade/main.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"os"
+)
+
+const mongoDBURL = "mongodb://user:password@mongodb:27017/keptn"
+
+var client *mongo.Client
+
+var projectCollections map[string]*mongo.Collection
+
+func main() {
```

Comment:
Can we please introduce arguments for passing the `mongoDBURL`? If provided use it, if not use the constant.
Reason: One customer used an external mongodb and potentially this upgrade script would also work for them.